src/commons_feval_feface.jl

```diff
@@ -0,0 +1,293 @@
+# Common methods for both FEValues and FEFaceValues
+typealias FEValuesOrFEFaceValues{dim,T,FS,GS} Union{FEValues{dim,T,FS,GS}, FEFaceValues{dim,T,FS,GS}}
```

Comment:
Name suggestion?
I wanna name it `FEValues` and rename the current `FEValues` to `FECellValues` and `FEFaceValues` to `FEBoundaryValues` or something